pkg/cmd/roachtest/tests/kv.go

```diff
@@ -36,8 +36,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const envKVFlags = "ROACHTEST_KV_FLAGS"
```

Comment:
Are these used in any CI jobs? If yes, we might want to migrate them also to the new flag